optaplanner-core/src/main/java/org/optaplanner/core/impl/heuristic/selector/common/nearby/BetaDistributionNearbyRandom.java

```diff
@@ -25,13 +25,13 @@
     protected final BetaDistribution betaDistribution;
 
     public BetaDistributionNearbyRandom(double betaDistributionAlpha, double betaDistributionBeta) {
-        if (betaDistributionAlpha < 0) {
+        if (betaDistributionAlpha <= 0) {
             throw new IllegalArgumentException("The betaDistributionAlpha (" + betaDistributionAlpha
-                    + ") must be at least 0.");
+                    + ") must be positive.");
```

Comment:
0 is positive, so the error message is not accurate.
Note: 0 is also negative. 0 is not strictly positive. 0 is also not strictly negative. https://proofwiki.org/wiki/Definition:Strictly_Positive

Changing into "must be greater than 0", to avoid confusion.